runtime/gc_modron_startup/mminit.cpp

```diff
@@ -1089,6 +1096,18 @@ gcInitializeXmxXmdxVerification(J9JavaVM *javaVM, IDATA* memoryParameters, bool
 		}
 	}
 
+#if defined(J9ZTPF)
+
+        usableMemory = omrsysinfo_get_addressable_physical_memory();
+
```

Comment:
The output of omrsysinfo_get_addressable_physical_memory() is stored in extensions->usablePhysicalMemory already. Please use it instead of Port Library call